Cargo.toml

```
@@ -24,39 +24,39 @@ client = ["substrate-subxt-client"]
 integration-tests = []
 
 [dependencies]
-log = "0.4.11"
-thiserror = "1.0.20"
-futures = "0.3.5"
+log = "0.4.13"
+thiserror = "1.0.23"
+futures = "0.3.10"
 jsonrpsee = { version = "0.1.0", features = ["ws"] }
-num-traits = { version = "0.2.12", default-features = false }
-serde = { version = "1.0.115", features = ["derive"] }
-serde_json = "1.0.57"
-url = "2.1.1"
-codec = { package = "parity-scale-codec", version = "1.3.5", default-features = false, features = ["derive", "full"] }
-
-frame-metadata = { version = "12", package = "frame-metadata" }
-frame-support = { version = "2.0.0", package = "frame-support" }
-sp-runtime = { version = "2.0.0", package = "sp-runtime" }
-sp-version = { version = "2.0.0", package = "sp-version" }
-pallet-indices = { version = "2.0.0", package = "pallet-indices" }
+num-traits = { version = "0.2.14", default-features = false }
+serde = { version = "1.0.119", features = ["derive"] }
+serde_json = "1.0.61"
+url = "2.2.0"
+codec = { package = "parity-scale-codec", version = "1.3.6", default-features = false, features = ["derive", "full"] }
+
+frame-metadata = { version = "12.0.1", package = "frame-metadata" }
```

not really introduced by this PR but can we remove the redundant "package" here and below?

```suggestion
frame-metadata = "12.0.1"
```

---

Sure, probably a hangover from when git deps used